File: backend/src/main/java/com/grupo6daw/lcdd_daw/dto/NewDTO.java
package com.grupo6daw.lcdd_daw.dto;

import java.time.LocalDateTime;
import java.util.List;

public record NewDTO(
        Long newId,
        String newName,
        String newDescription,
        ImageDTO newImage,
        String newTag,
        List<EventBasicDTO> newEvents,
        UserBasicDTO newCreator,
        boolean validated,
        LocalDateTime creationDate) {
}

File: frontend/app/dtos/NewDTO.ts
import type EventBasicDTO from "./EventBasicDTO";
import type ImageDTO from "./ImageDTO";
import type UserBasicDTO from "./UserBasicDTO";

export default interface NewDTO {
  newId: number;
  newName: string;
  newDescription: string;
  newImage: ImageDTO;
  newTag: string;
  newEvents: EventBasicDTO[];
  newCreator: UserBasicDTO;
  validated: boolean;
  creationDate: string;
}